src/coreclr/gc/gc.cpp

```diff
@@ -3571,7 +3571,7 @@ uint8_t* region_allocator::allocate (uint32_t num_units)
     uint32_t* current_index = region_map_start;
     uint32_t* end_index = region_map_end;
 
-    dprintf (1, ("\nsearcing %d->%d", (int)(current_index - region_map_start), (int)(end_index - region_map_start)));
+    dprintf (1, ("searching %d->%d", (int)(current_index - region_map_start), (int)(end_index - region_map_start)));
```

Comment:
Should the dprintf level be REGIONS_LOG here?

Reply:
I will change all of the dprintf's in region_allocator to REGIONS_LOG.